pkg/metrics/metrics.go

```diff
@@ -123,16 +124,21 @@ func New(conf *conf.Main) *Prom {
 			Name:      "udp_read_failures_total",
 			Help:      "Counter of failures when reading incoming data from the UDP connection.",
 		}),
+		RegexDuration: prometheus.NewSummaryVec(prometheus.SummaryOpts{
+			Namespace: "nanotube",
+			Name:      "regex_duration_seconds",
+			Help:      "Time to evaluate each regex from configuration",
+		}, []string{"regex", "rule_type"}),
 		Version: prometheus.NewCounterVec(prometheus.CounterOpts{
 			Namespace: "nanotube",
 			Name:      "version",
 			Help:      "Version info in label. Value should be always 1.",
 		}, []string{"version"}),
-		RegexDuration: prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		ConfVersion: prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Random thought: if we make it a gauge, we can store the hash as the value. 
Not sure if using the gauge would be easier, harder or the same...

Reply:
Thx for the idea.